keps/sig-node/compute-device-assignment.md

```diff
@@ -57,15 +59,33 @@ In this document we will discuss the motivation and code changes required for in
 
 ![device monitoring architecture](https://user-images.githubusercontent.com/3262098/43926483-44331496-9bdf-11e8-82a0-14b47583b103.png)
 
+### Device aware CNI plugin
+As soon as this interface has been introduced it was used by CNI plugins like [kuryr-kubernetes](https://review.opendev.org/#/c/651580/) in couple with [intel-sriov-device-plugin](https://github.com/intel/sriov-network-device-plugin) to correctly define which devices were assigned to the pod.
 
-## Changes
+### Topology aware scheduling
+This interface can be used to collect allocated resources with information about the NUMA topology of the worker node.
+This interface can be used to the available resources on the worker node. The kubelet is the best source of information because it manages the device plugins. The information can then be used in NUMA aware scheduling.
 
-Add a v1alpha1 Kubelet GRPC service, at `/var/lib/kubelet/pod-resources/kubelet.sock`, which returns information about the kubelet's assignment of devices to containers. It obtains this information from the internal state of the kubelet's Device Manager. The GRPC Service returns a single PodResourcesResponse, which is shown in proto below:
+## Changes
+Add a v1alpha1 Kubelet GRPC service, at `/var/lib/kubelet/pod-resources/kubelet.sock`, which returns information about
+- the devices which kubelet knows about from the device plugins.
+- the kubelet's assignment of devices and cpus with NUMA id to containers.
+The GRPC service obtains this information from the internal state of the kubelet's Device Manager and CPU Manager respectively.
+The GRPC Service is described in proto below:
 ```protobuf
 // PodResources is a service provided by the kubelet that provides information about the
 // node resources consumed by pods and containers on the node
 service PodResources {
     rpc List(ListPodResourcesRequest) returns (ListPodResourcesResponse) {}
+    rpc GetAvailableResources(AvailableResourcesRequest) returns (AvailableResourcesResponse) {}
```

Comment:
maybe change this to `GetAllocatableResources` to align with node allocatable.

Reply:
how often do you anticipate clients invoking this?  for things like hugepages or devices (where the plugin is deployed async or dynamically updated), i assume clients will just poll every X interval?

Reply:
GetAvailableResources (or GetAllocatableResources) will be called once the topology exporter is starting on the worker node.
kubelet reconfiguration like changing CPUManager policy will require recalling GetAllocatableResources.

Reply:
@AlexeyPerevalov answered already; in general we indeed expect this API to be invoked rarely, pretty much once per client, more times only if kubelet restarts.